### ISSUE TITLE
Gov Accessibility Guidance Document template for client work

### ISSUE THREAD

**Issue by xtine** [trussworks/accessibility#13]
I currently have a Accessibility Guidance Document template for government(-adjacent) clients that's currently in GDrive that should be also included in this repo. It is sharable introduction to accessibility and what Section 508 compliance means for project work.
